packages/quip-cli/src/commands/publish.ts

```diff
@@ -92,25 +92,26 @@ export const doPublish = async (
         ignore
     );
     if (missing.size > 0) {
-        println(chalk`{red WARNING: the following files were defined in your manifest, but were not found.}
+        println(chalk`{yellow WARNING: the following files were defined in your manifest, but were not found.}
 {red This bundle may be incomplete, you should include these files or remove them from your manifest.}`);
         for (let [source, files] of missing) {
-            println(chalk`{red === ${source} ===}`);
-            files.forEach(f => println(chalk`{red ${f}}`));
+            println(chalk`{yellow === ${source} ===}`);
+            files.forEach((f) => println(chalk`{yellow ${f}}`));
         }
+        println(
+            chalk`{green Note: If only app.js is missing, please try to compile the live app by running \`npm run build\` and try publishing again.}`
```

Comment:
Hi @abbyomotoso , I would like to get some feedback from you about this developer-facing message from the terminal. 

I am not sure whether we need to disclose it in the developer guide. 
(If we address it, of course, developer will have less surprise. As the behavior of `quip-cli publish` get changed with this PR)

Let's see if there is any concern from @nzapponi .

Reply:
@abbyomotoso @zhouhao I don't think it's necessary to add documentation about this beyond the CLI message.

Reply:
> 
> 
> Hi @abbyomotoso , you are right, there are no customer-facing docs involved in this PR yet. This PR tries to fix [quip/issues#20159](https://github.com/quip/issues/issues/20159).
> 
> And I would like to get some feedback from you about this developer-facing message from the terminal.
> 
> I am not sure whether we need to disclose it in the developer guide. (If we address it, of course, developer will have less surprise. As the behavior of `quip-cli publish` get changed with this PR)
> 
> Let's see if there is any concern from @nzapponi .

Hi @zhouhao . Can I get back to about the developer-facing message on 11/24? I may have some questions for you before then.

Reply:
Thank you @nzapponi . Sure, @abbyomotoso . Please feel free to reach me if you have any questions or concerns.

Reply:
Hi @zhouhao . Will app.js always be missing if a developer tries to publish a live app without compiling it first?

Reply:
Yes and Yes!

app.js is generated automatically by executing `npm run build`. Without aap.js, the live app cannot be published(publishing will fail, and you can find more details from above screenshot).